src/utils/NetworkUtils.ts

```diff
@@ -156,6 +156,7 @@ export function chainIsCCTPEnabled(chainId: number): boolean {
     CHAIN_IDs.OPTIMISM,
     CHAIN_IDs.POLYGON,
     CHAIN_IDs.UNICHAIN,
+    CHAIN_IDs.LINEA,
```

Comment:
wdyt about this? https://github.com/across-protocol/sdk/pull/959

It should be OK to use it without adding Linea to `cctpExceptions` because we've disabled Linea for USDC.